Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM node:carbon as builder
+FROM node:carbon-alpine as builder
+RUN apk --no-cache add python build-base
```

Comment:
should remove them after the rebuild

Reply:
According to the docoumentation, we don't care:
``` The Go SDK and any intermediate artifacts are left behind, and not saved in the final image.```
https://docs.docker.com/engine/userguide/eng-image/multistage-build/#use-multi-stage-builds

Reply:
Also, this is just the builder image to copy stuff from, so we don't care too much about the cleanliness of the image afterwards.

Reply:
you are correct actually  which is why this isnt needed as its already on line 10
still shoudl be removed

Reply:
It's needed for when `yarn` runs in the builder.

Reply:
build-base & python should already be in `node:carbon`